packages/SwingSet/misc-tools/measure-metering/measure.js

```diff
@@ -139,8 +134,9 @@ async function run() {
   const zoeInstallVaultFactory = await doCounted('zoeInstallVaultFactory');
   console.log(`zoe install (vaultFactory): ${zoeInstallVaultFactory}`);
 
-  const zoeInstallAMM = await doCounted('zoeInstallBundle', [autoswapBundle]);
-  console.log(`zoe install (AMM): ${zoeInstallAMM}`);
+  // XXX  Does this need to be replace by something else in order to be useful?
```

Comment:
I don't think so. Who uses this tool… maybe @warner ?

Reply:
Don't worry about it.

The misc-tools directory has a `README-analysis-tools.md` disclaimer:

> They were built as-needed to investigate various performance issues, and vary drastically in quality and usability.

I think the PSM is a likely replacement, but I think it's fair to leave it in XXX state until somebody comes along and wants to use it again.